validator/accounts/accounts_import_test.go

```diff
@@ -131,33 +131,6 @@ func TestImport_DuplicateKeys(t *testing.T) {
 	assert.Equal(t, 1, len(keys))
 }
 
-// TestImport_NonImportedWallet is a regression test that ensures non-silent failure when importing to non-imported wallets
```

Comment:
Derived wallets should be able to import keystores as they are merely wrappers over imported wallets